v1/src/main/java/com/google/cloud/teleport/templates/common/DatadogConverters.java

```diff
@@ -181,7 +173,7 @@ public interface DatadogOptions extends PipelineOptions {
     void setApiKeySecretId(ValueProvider<String> secretId);
 
     @TemplateParameter.Enum(
-        order = 9,
+        order = 8,
```

Comment:
These were typos from the initial contribution, not related to the parameter change.